shared/message/motion/KinematicsModel.proto

```diff
@@ -108,10 +136,12 @@ message KinematicsModel {
         mat3 foot        = 9;
     }
 
-    Leg         leg                       = 1;
-    Head        head                      = 2;
-    Arm         arm                       = 3;
-    MassModel   massModel                 = 4;
-    TensorModel tensorModel               = 5;
-    float       TEAMDARWINCHEST_TO_ORIGIN = 6;
+    // Instantiations of the config containers.
```

Comment:
Comment format should be consistent

Reply:
hmm, I had this one different because it described the group of variables, rather than the single variable. Is there a better way to indicate this?